hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -44,7 +44,7 @@ import { RichTextEditor } from '@mantine/rte'
 type IProps = {
   hackathonId: string
   type: HackathonDetailsType
-  reloadList?: () => void
+  reloadList: () => void
```

Comment:
same thing about the name here, seems to only be used when a hackathon is deleted

Reply:
here, it also used for when a hackathon is edited. 
What proper name to used for change and delete?
`onChanged`(?)

Reply:
`onSuccess` seems good enough